### PR TITLE
🚀 refactor(chips): adjust chips texts 

### PR DESCRIPTION
**Pull request by CaioAugustoR**
[JIRA Issue](https://gympass.atlassian.net/browse/)

## Description 📄

This PR implements the update of text nomenclatures in chips, in accordance with the guidelines of the new rebranding. The changes were applied as follows:

- For chips in the `selected` or `disabled` state, the text style was changed from `Text.Bold` to `Text.Overline`.
- For chips in the default state, the text style was updated from `Text` to `Text.Caption`.

These changes ensure that the use of chips is aligned with the new visual identity and improves consistency across the user interface.

## Platforms 📲

- [x] Web
- [x] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [ ] Snapshot Test

## Checklist: 🔍

- [ ] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [x] Layout matches design prototype: [FIGMA](https://www.figma.com/file/Hsrgmg4oLjJxYcJHaFeuJP/%F0%9F%8E%A8-Yoga-3?type=design&node-id=51-2839&mode=design&t=8eG1HEdhvAUluXgi-0)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Screenshots 📸

| Before | After |
| ------ | ----- |
|Simulator Screenshot - iPhone 15 Pro - 2024-03-18 at 10 50 44|Simulator Screenshot - iPhone 15 Pro - 2024-03-18 at 10 49 53|
|Simulator Screenshot - iPhone 15 Pro - 2024-03-18 at 10 50 48|Simulator Screenshot - iPhone 15 Pro - 2024-03-18 at 10 49 56|

| Before | After |
| ------ | ----- |
|Captura de Tela 2024-03-18 às 10 56 29|Captura de Tela 2024-03-18 às 10 55 18|
|Captura de Tela 2024-03-18 às 10 56 46|Captura de Tela 2024-03-18 às 10 55 37|
|Captura de Tela 2024-03-18 às 10 56 56|Captura de Tela 2024-03-18 às 10 56 01|

### CODE REVIEW

**Review by joaopaulolimat — APPROVED** [review]

File: packages/yoga/src/Chips/web/Chips.jsx [hunk 1]
```diff
@@ -173,7 +156,11 @@ const Chips = React.forwardRef(
             }}
           />
         )}
-        {children && <Text>{children}</Text>}
+        {children && (selected || disabled) ? (
```
Comment (leonardo-gympass) [inline packages/yoga/src/Chips/web/Chips.jsx:159]: Shouldn't we display the `Text.Caption` only when the `children` exist to maintain consistency with previous behavior?

Comment (CaioAugustoR) [reply packages/yoga/src/Chips/web/Chips.jsx:159]: Yes, that's what happens in this piece of code, right?

Comment (leonardo-gympass) [reply packages/yoga/src/Chips/web/Chips.jsx:159]: The `Text.Caption` will be rendered when the condition is `false`.
Screenshot 2024-03-19 at 08 51 05

We could create a variable to define the text component.
```jsx
const TextComponent = selected || disabled ? Text.Overline : Text.Caption;

children && <TextComponent>{children}</TextComponent>
```

Comment (CaioAugustoR) [reply packages/yoga/src/Chips/web/Chips.jsx:159]: You're right, I'll work on the change!

**Review by leonardo-gympass — APPROVED** [review]

**Review by thiamancio — APPROVED** [review]

**Review by ericcleao — APPROVED** [review]

**Review by flavia-moraes — APPROVED** [review]

**Review by rafaelcoletagympass — APPROVED** [review]

**Pull request merged**